apps/system-e2e/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#
+# This script is used to build the system-e2e app using esbuild.
+# Playwright requieres special handling with esbuild, which is why it can't use
+# the "normal" build process.
+#
```

Comment:
Even this seems to be failing when running against the built output. Playwright can't find our `expect` extensions like `expect.toHaveCountGreaterThan(...)`. It works locally, however, both when tested directly and with the `--ui` flag.

Reply:
Removed all custom matchers :heavy_check_mark: